iroh-base/src/lib.rs

```diff
@@ -1,23 +1,45 @@
 //! Base types and utilities for Iroh
 #![cfg_attr(iroh_docsrs, feature(doc_cfg))]
 
+// TODO: remove
 #[cfg(feature = "base32")]
 #[cfg_attr(iroh_docsrs, doc(cfg(feature = "base32")))]
 pub mod base32;
+
+// TODO: move to own crate
+#[cfg(feature = "base32")]
+#[cfg_attr(iroh_docsrs, doc(cfg(feature = "base32")))]
+pub mod ticket;
+
 #[cfg(feature = "hash")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "hash")))]
-pub mod hash;
+mod hash;
 #[cfg(feature = "key")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "key")))]
-pub mod key;
+mod key;
 #[cfg(feature = "key")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "key")))]
-pub mod node_addr;
+mod node_addr;
 #[cfg(feature = "relay")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "relay")))]
-pub mod relay_map;
+mod relay_map;
 #[cfg(any(feature = "relay", feature = "key"))]
 mod relay_url;
-#[cfg(feature = "base32")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "base32")))]
-pub mod ticket;
+
+#[cfg(feature = "relay")]
+#[cfg_attr(iroh_docsrs, doc(cfg(feature = "relay")))]
+pub use self::relay_map::{
+    QuicConfig as RelayQuicConfig, RelayMap, RelayNode, DEFAULT_RELAY_QUIC_PORT, DEFAULT_STUN_PORT,
```

Comment:
Not sure I like having these constants at top level. It is probably very rare that somebody needs them, so having them hidden away in some public mod is not that bad.

Reply:
I didn't find a better way, this is only for `iroh-base` though, in `iroh` it is much cleaner now